Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.9-slim
+
+# Set pip to have cleaner logs and no saved cache
+ENV PIP_NO_CACHE_DIR=false
+    # POETRY_VIRTUALENVS_CREATE=false
+
+# Install Poetry
+RUN pip install --upgrade poetry
+
+
+# Create the working directory
+WORKDIR /modmail
+
+# Install project dependencies
+
+# Copy dependencies and lockfile
+COPY pyproject.toml poetry.lock ./
+RUN poetry install --no-dev --no-interaction --no-ansi --no-root
+
+# Copy the source code in last to optimize rebuilding the image
+COPY . .
+
+ENTRYPOINT ["poetry"]
```

Comment:
I have never done this on Dockerfiles in my past projects, don't see how this is needed here, actually, I don't see why this is not needed here too 😅. Mind explaining?

Reply:
Due to line 5 causing issues, the entry point on my machine had to be into the venv. This shouldn't be a problem, but it somehow is. I think it may be because some of our dependencies do not agree with poetry.

Reply:
Uhhh okay, am still not clear. I will look around the docs and ask joe, if I understand the issue in a better way.